toolkit/imageconfigs/linuxguard-amd64.yaml

```diff
@@ -92,14 +92,14 @@ storage:
       mountPoint:
         idType: part-label
         path: /var/lib/trident
-        options: nodev
+        options: noexec,nodev
 
     - deviceId: home
       type: ext4
       mountPoint:
         idType: part-label
         path: /home
-        options: nodev
+        options: noexec,nodev
 
 os:
   bootloader:
```

Comment:
Should we change the hostname? E.g. azure-linux-guard?

Reply:
updated